File: samples/tagging/python-individual/upload-youtube-link.py
import requests
from requests.auth import HTTPBasicAuth
import json

api_key = ""    # Your API Key
youtube_link = "" # Your YouTube link

url = "https://api-us.musiio.com/api/v1/upload/youtube-link"

payload = {
    "link": youtube_link
}

headers = {
  'Content-Type': 'application/json'
}

response = requests.post(url, headers=headers, data=json.dumps(payload), auth=HTTPBasicAuth(api_key, ''))

print(response.text.encode('utf8'))

File: samples/tagging/node/upload-youtube-link.js
const YOUTUBE_LINK = "";  // Youtube link here
const API_KEY = "";       // API key here

var axios  = require('axios');
var data = JSON.stringify({"link": YOUTUBE_LINK});

var config = {
  method: 'post',
  url: 'https://api-us.musiio.com/api/v1/upload/youtube-link',
  auth: {
    username: API_KEY,
    password: ""
  },
  headers: { 
    	'Content-Type': 'application/json'
  },
  data : data
};

axios(config)
.then(function (response) {
  console.log(JSON.stringify(response.data));
})
.catch(function (error) {
  console.log(error);
});

